src/creatures/combat/spells.cpp

```diff
@@ -630,6 +630,40 @@ void Spell::setWheelOfDestinyBoost(WheelSpellBoost_t boost, WheelSpellGrade_t gr
 	}
 }
 
+void Spell::getCombatDataAugment(std::shared_ptr<Player> player, CombatDamage &damage) {
+	if (!(damage.instantSpellName).empty()) {
+		std::vector<std::shared_ptr<Item>> equippedAugmentItems = player->getEquippedAugmentItems();
+		for (std::shared_ptr<Item> item : equippedAugmentItems) {
+			Augments augments = item->getAugmentsBySpellName(damage.instantSpellName);
+			for (auto &augment : augments) {
+				if (augment->type == AUGMENT_INCREASED_DAMAGE || augment->type == AUGMENT_POWERFUL_IMPACT || augment->type == AUGMENT_STRONG_IMPACT) {
+					const float augmentPercent = augment->value / 100.0;
```

Comment:
Divisions by 0 can cause undefined behavior (crash) it may be interesting to check that "augment->value" is not 0 before dividing

```suggestion
					const float augmentPercent = augment->value ? augment->value / 100.0 : 0;
```